.github/workflows/cmake.yml

```diff
@@ -79,6 +79,48 @@ jobs:
           diff -u src/obj/build.h build/src/obj/build.h
 
 
+  ubuntu-focal-native:
+    name: 'Ubuntu 20.04, CMake 3.16, Boost from depends'
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Remove preinstalled CMake
+        run: |
+          sudo rm /usr/local/bin/cmake /usr/local/bin/ctest
```

Comment:
Is this something that comes with the github actions image? I do not have this package when I do:
```
docker pull ubuntu:20.04
docker run -ti --rm ubuntu:20.04 /bin/bash
```

Reply:
> Is this something that comes with the github actions image?

It is.